tests/integration/api_container_test.py

```diff
@@ -809,6 +810,25 @@ def test_logs_with_tail_0(self):
         logs = self.client.logs(id, tail=0)
         self.assertEqual(logs, ''.encode(encoding='ascii'))
 
+    def test_logs_with_since_kwarg(self):
```

Comment:
This might make more sense as a unit test, since we really just want to check that the method raises an error when passed invalid arguments, creating an actual container or connecting to the Docker Engine is unnecessary.